src/CADL.Extension/Emitter.Csharp/src/lib/cadlServer.ts

```diff
@@ -39,7 +39,7 @@ function getDefaultValue(type: Type): any {
 
 export function resolveServers(
     program: Program,
-    servers: http.HttpServer[],
+    servers: HttpServer[],
     models: Map<string, InputModelType>,
     enums: Map<string, InputEnumType>
 ): CadlServer[] {
```

Comment:
Do we still need 'CadlServer' ? 
And is it possible to rename this file as well?

Reply:
We have not update the source code. we will update in another PR together with the project folder name